src/sb_gazebo/urdf/jfrost.gazebo

```diff
@@ -4,95 +4,108 @@
 -->
 <robot name="jfrost" xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-    <gazebo reference="chassis">
-      <material>Gazebo/Blue</material>
-    </gazebo>
+  <gazebo reference="chassis">
+    <material>Gazebo/Blue</material>
+  </gazebo>
 
-    <gazebo>
-        <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
-            <robotNamespace>/robot</robotNamespace>
-        </plugin>
-    </gazebo>
+  <gazebo>
+    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+      <robotNamespace>/robot</robotNamespace>
+    </plugin>
+  </gazebo>
 
-    <!-- STEERING CONTROLLER -->
+  <!-- STEERING CONTROLLER -->
 
-    <!-- DIFFERENTIAL DRIVE -->
-    <gazebo>
-      <plugin name="differential_drive_controller" filename="libgazebo_ros_diff_drive.so">
+  <!-- DIFFERENTIAL DRIVE -->
+  <gazebo>
+    <plugin name="differential_drive_controller" filename="libgazebo_ros_diff_drive.so">
+      <alwaysOn>true</alwaysOn>
+      <updateRate>50</updateRate>
+      <leftJoint>left_front_wheel_hinge</leftJoint>
+      <rightJoint>right_front_wheel_hinge</rightJoint>
+      <wheelSeparation>${chassis_width+wheel_width}</wheelSeparation>
+      <wheelDiameter>${2*wheel_radius}</wheelDiameter>
+      <wheelTorque>3</wheelTorque>
+      <commandTopic>robot/cmd_vel</commandTopic>
+      <odometryTopic>robot/odom_diffdrive</odometryTopic>
+      <odometryFrame>odom</odometryFrame>
+      <robotBaseFrame>base_link</robotBaseFrame>
+      <legacyMode>False</legacyMode>
+    </plugin>
+  </gazebo>
+
+  <!-- IMU -->
+  <gazebo>
+    <plugin name="imu_controller" filename="libgazebo_ros_imu.so">
+      <alwaysOn>true</alwaysOn>
+      <updateRate>50.0</updateRate>
+      <bodyName>chassis</bodyName>
+      <topicName>/imu</topicName>
+      <gaussianNoise>2.89e-08</gaussianNoise>
+      <xyzOffsets>0 0 0</xyzOffsets>
+      <rpyOffsets>0 0 0</rpyOffsets>
+    </plugin>
+  </gazebo>
+
+  <!-- Zed camera -->
+  <gazebo reference="camera">
+    <sensor type="depth" name="zed_camera">
+      <always_on>1</always_on>
+      <update_rate>30.0</update_rate>
+      <visualize>true</visualize>
+      <camera>
+        <horizontal_fov>1.047</horizontal_fov>
```

Comment:
@RobynCastro - mentioning you here so you are notified when this change is merged. Probably want to do some tests with vision to make sure everything is still happy.